lib/index.js

```diff
@@ -4,13 +4,21 @@ const path = require('path');
 const BinWrapper = require('bin-wrapper');
 const pkg = require('../package.json');
 
-const url = `https://raw.github.com/1000ch/simplehttp2server/v${pkg.version}/vendor/`;
+const url = `https://github.com/GoogleChrome/simplehttp2server/releases/download/${pkg.version}/`;
```

Comment:
Should be `https://github.com/GoogleChrome/simplehttp2server/releases/download/3.1.3/`?